lib/ProductOpener/Mail.pm

```diff
@@ -160,6 +162,75 @@ sub send_email ($user_ref, $subject, $text) {
 	return _send_email($mail);
 }
 
+=head2 send_html_email( USER_REF, SUBJECT, HTML_CONTENT )
+
+C<send_html_email()> sends an HTML email from the contact email of Open Food Facts to the email passed as an argument.
+
+=head3 Arguments
+
+The first argument is a hash reference. The other two arguments are scalar variables that consist of email subject and HTML content.
+
+=head4 Input keys: user data
+
+The hash must contain values for the following keys:
+
+- email -> user email
+- name -> user name
+
+=head3 Return Values
+
+The function returns a 1 or 0 depending on the evaluation of the email sent or not.
+
+If the function catches any error during evaluation it returns 1 indicating an error.
+On the other hand, if there was no error, it returns 0 indicating that the email has been sent successfully.
+
+=cut
+
+sub send_html_email ($user_ref, $subject, $html_content) {
+	my $email = $user_ref->{email};
+	my $name = $user_ref->{name};
+
+	my $mail = Email::Stuffer->from(lang("site_name") . " <$contact_email>")->to($name . " <$email>")->subject($subject)
+		->html_body($html_content);
+	return _send_email($mail);
+}
+
+=head2 get_html_email_content ( FILENAME, LANG )
+
+Fetch the HTML email content in $DATA_ROOT/lang/emails. If a translation is available
+for the requested language, we provide the translated version otherwise English is the default.
+
+
+=head3 Arguments
+
+=head4 FILENAME
+
+The HTML file name (ex: user_welcome.html)
+
+=head4 LANG
+
```

Comment:
You must keep constistent with real definition

```suggestion
=head2 get_html_email_content ($filename, $lang )

Fetch the HTML email content in $DATA_ROOT/lang/emails. If a translation is available
for the requested language, we provide the translated version otherwise English is the default.


=head3 Arguments

=head4 $filename

The HTML file name (ex: user_welcome.html)

=head4 $lang

```

Reply:
Alright, I took example from another docstring :sweat_smile: